recipes/hdmf/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "hdmf" %}
+{% set version = "0.post0.dev1081" %}
+{% set sha256 = "8b0264a51ef1aea5fb22d450a7fa6cbc74bac97421e9667d1b7caee0ab74ce57" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - python
+    - pip
+    - setuptools
+  host:
+    - pip
```

Comment:
I think you need python and setuptools here, and not in the build section

Reply:
done